sabnzbd/deobfuscate_filenames.py

```diff
@@ -118,11 +118,21 @@ def is_probably_obfuscated(myinputfilename):
     return True  # default not obfuscated
 
 
-def deobfuscate(workingdirectory, usefulname):
-    """ In workingdirectory, check all filenames, and if wanted, rename """
+def deobfuscate_dir(workingdirectory, usefulname):
+    """ In workingdirectory, check all filenames, and if wanted, deobfuscate """
+
+    list_of_files = []
+    for (dirpath, dirnames, filenames) in os.walk(workingdirectory):
+        list_of_files += [os.path.join(dirpath, file) for file in filenames]
```

Comment:
Let's just use the `globber_full` function, it does exactly this and is already imported! See deleted line below. Just leave out the pattern and it should list then all.